DEVELOPERS.md

```diff
@@ -54,7 +66,7 @@ just clean
 just test-outputs
 ```
 
-## Running the electon app
+## Running the electon app in dev mode
```

Comment:
In this section, you could change:
`You can run the development version the python webapp and then run the electron`
To
`First, run the development version of the python webapp and then run the electron`